scripts/lua/management/lib/tenants.lua

```diff
@@ -114,7 +114,39 @@ function _M.getTenantAPIs(dataStore, id, queryParams)
       end
     end
   end
-  return apiList
+  if ((queryParams['skip']  == nil and queryParams['limit'] == nil) or table.getn(apiList) == 0) then
+    return apiList
+  else
+    return applyPagingToAPIs(apiList, queryParams)
+  end
+end
+
+-- Apply paging on apis
+-- @param apis the list of apis
+-- @param queryparams object containing optional query parameters
+function applyPagingToAPIs(apiList, queryParams)
+  local skip  = queryParams['skip']  == nil and 1 or queryParams['skip']
+  local limit = queryParams['limit'] == nil and table.getn(apiList) or queryParams['limit']
+  if (tonumber(limit) < 1) then
+    return {}
+  end
+  if (tonumber(skip) <= 0) then
+    skip = 1
+  else
+    skip = skip + 1
+  end
+  if ((skip + limit - 1) > table.getn(apiList)) then
+    limit = table.getn(apiList)
+  else
+    limit = skip + limit - 1
+  end
+  local apis  = {}
+  local idx   = 0
+  for i = skip, limit do
```

Comment:
i'm not fluent in lua and did not RTM - is `for i = skip, limit` ranging from [1..n] inclusive when the `skip` query param is not defined?

Reply:
The line below takes care of that:
`local skip  = queryParams['skip']  == nil and 1 or queryParams['skip']`
This line reads: If the skip parameter is undefined (nil), assign the value 1 to the skip variable, otherwise use the value of the skip parameter.